src/admin/class-sign-in-with-google-admin.php

```diff
@@ -364,6 +373,17 @@ public function siwg_show_on_login() {
 
 	}
 
+	/**
+	 * Callback function for Show Google Signup Button on Login Form
+	 *
+	 * @since    1.0.0
```

Comment:
```suggestion
	 * @since    [NEXT]
```
I'll do a find/replace when prepping the release so we know exactly what version these methods are introduced in.

Reply:
yes, I've placed placeholders, and even missed to set them correctly. i've left them for you,